kentik_api_library/kentik_api/requests_payload/conversions.py

```diff
@@ -49,26 +49,27 @@ def process_list(l: List[Any]) -> List[Any]:
 
 
 def from_dict(data_class: Type[T], data: Data) -> T:
-    """ Converts given dictionary to the data class of given type. It converts Dacite errors into DeserializationError """
+    """Converts given dictionary to the data class of given type. It converts Dacite errors into
+    DeserializationError"""
 
     try:
         return dacite.from_dict(data_class=data_class, data=data)
     except dacite.DaciteError as err:
-        raise DeserializationError(data_class.__name__, str(err))
+        raise DeserializationError(data_class.__name__, str(err)) from err
```

Comment:
I think "from err" here is unnecessary. According to  https://docs.python.org/3/tutorial/errors.html#exception-chaining
"Exception chaining happens automatically when an exception is raised inside an except or finally section."

Reply:
By default `__context__` is set. If `from err` is used, then `__cause__` is set. Messages differ in traceback and when `from err` is used, then you can see that "DaciteError is a direct cause of DeserializationError" or something like that. In other way you can see something like "exception DeserializationError raised during handling of DaciteError" 
IMO using "from err" is a better approach for error translation.

See https://docs.python.org/3/library/exceptions.html#bltin-exceptions